src/Evolution/Systems/Cce/Actions/InitializeWorldtubeBoundary.hpp

```diff
@@ -170,5 +170,44 @@ struct InitializeWorldtubeBoundary<GhWorldtubeBoundary<Metavariables>>
   using typename BaseType::initialization_tags_to_keep;
 };
 
+/*!
+ * \ingroup ActionsGroup
+ * \brief Initializes an AnalyticWorldtubeBoundary
+ *
+ * \details Uses:
+ * - initialization tag
+ * `Cce::Tags::AnalyticBoundaryDataManager`,
+ * - const global cache tags `Tags::LMax`,
+ * `Tags::ExtractionRadius`.
+ *
+ * Databox changes:
+ * - Adds:
+ *   - `Tags::Variables<typename
+ * Metavariables::cce_boundary_communication_tags>`
```

Comment:
extra indentation in bullet. (also did you render the documentation and look?)

Reply:
Sorry, I think Doxygen renders it correctly, I find it easier when I read the comments in a editor :P